cmd/cstor-volume-mgmt/volume/volume_test.go

```diff
@@ -21,10 +21,18 @@ import (
 
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/util"
+	"k8s.io/apimachinery/pkg/api/resource"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+//TODO: Move to some common function
+func StrToQuantity(capacity string) resource.Quantity {
```

Comment:
I fear moving this to some common func. We are swallowing the error here. This might be ok in UT but not in actual source code.

Reply:
Since it is UT intentionally swallowing error.